pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -961,10 +961,6 @@ func (og *operationGenerator) GenerateUnmapVolumeFunc(
 		// Try to unmap podUID symlink under global map path dir
 		// plugins/kubernetes.io/{PluginName}/volumeDevices/{volumePluginDependentPath}/{podUID}
 		globalUnmapPath := volumeToUnmount.DeviceMountPath
```

Comment:
On reconstruction, how does this get set?

Reply:
I separated this problem for non-attachable plugin and volume reconstruction to recover local volume plugin quickly.
During reconstruction path, if the volumeMode is `Block`, kubelet can't find  globalUnmapPath without volumeSpec. Therefore, I think we should return error during `cleanupMounts()` for Block volume case.